src/gso.ts

```diff
@@ -190,15 +191,15 @@ export class GSO {
     );
     const so = new StakingOptions(this.connection.rpcEndpoint);
 
-    const soState = await so.state(`GSO${projectName}`, baseMint);
-    const soBaseVault = await so.baseVault(`GSO${projectName}`, baseMint);
-    const soOptionMint = await so.soMint(strikeAtomsPerLot, `GSO${projectName}`, baseMint);
+    const soState = await so.state(`GSO${projectName}`, soBaseMint);
+    const soBaseVault = await so.baseVault(`GSO${projectName}`, soBaseMint);
+    const soOptionMint = await so.soMint(strikeAtomsPerLot, `GSO${projectName}`, soBaseMint);
     const xBaseMint = await this.xBaseMint(gsoState);
     const baseVault = await this.baseVault(gsoState);
 
     // TODO: Init fee account if needed.
 
-    return this.program.instruction.config(
+    return this.program.instruction.configV2(
```

Comment:
no benefit to leaving the old one in correct?

Reply:
correct. leaving program support in case anyone is using the old sdk, but no need to support that ix in new client code